app/src/styles.sass

```diff
@@ -202,4 +203,3 @@ div.mat-menu-panel.nav-menu
   border: none
   height: 1px
   width: 78%
-  
```

Comment:
There is a convention in coding to end all text files with a new line. There is certainly a configuration option in your text editor to force it to respect the convention. Here the newline is missing.